test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -585,7 +585,7 @@ func gatherInternalAPIs(discoveryClient discovery.DiscoveryInterface, t *testing
 	for _, apiResourcesList := range apiResourcesLists {
 		gv, err := schema.ParseGroupVersion(apiResourcesList.GroupVersion)
 		if err != nil {
-			t.Errorf("error parsing group version %v, err: %v", apiResourcesList.GroupVersion, err)
+			t.Logf("error parsing group version %v, err: %v", apiResourcesList.GroupVersion, err)
```

Comment:
I don't think this change is correct - we want the failure if there is an error

Reply:
Should we remove the continue statement then? I don't think it serves a purpose if we're failing the test.

Reply:
You're right on this one, my bad. Can change to require.NoError instead of an if check